### ISSUE TITLE
Updating adds NWI files as droplets

### ISSUE THREAD

**Issue by instantflorian** [WBCE/News-with-Images#99]
After updating NWI to the current nightly build in an existing WBCE installation, the module files appear as droplets
![grafik](https://github.com/user-attachments/assets/ba5ff243-eef4-439b-bdc7-d082840056b3)
inside of them: the usual code of the module files
![grafik](https://github.com/user-attachments/assets/1c5dec81-665c-4fc8-b432-10664abb7717)

**Comment by mrbaseman** [comment]
I've just tried to reproduce this, but didn't get the described effect.
I have downloaded [News-with-Images-main.zip](https://github.com/WBCE/News-with-Images/archive/refs/heads/main.zip) and installed it in WBCE 1.6.2 which comes with Droplets 2.4.0.

I first had to uninstall NWI in order to be able to run the module install. Since I could not reproduce it this way, I have uninstalled it again, installed [NWI 5.0.25](https://github.com/WBCE/News-with-Images/releases/tag/5.0.25) (still no obsure droplets), and upgraded to the latest main of NWI, but couldn't reproduce it either.

Did you do a `git clone` and pack the zip manually? Or sync the files and run the install.php manually? Maybe something went wrong and by mistake the module files were contained a second time inside its droplets folder during one of these procedures?
